inspirehep/alembic/0bc0a6ee1bc0_create_index_on_referenced_records.py

```diff
@@ -0,0 +1,56 @@
+#
+# This file is part of Invenio.
+# Copyright (C) 2016-2018 CERN.
+#
+# Invenio is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""create index on referenced_records"""
+
+from alembic import op
+import sqlalchemy as sa
+
+# revision identifiers, used by Alembic.
+revision = '0bc0a6ee1bc0'
+down_revision = '2f5368ff6d20'
```

Comment:
hmm down revision should be the HEAD which I think is: ``53e8594bc789``

Reply:
I think it's ok:
```
dbdbc1b19cf2 -> 35c1075e6360 (default) (effective head), Force naming convention.
720ddf51e24b -> a26f133d42a9 (invenio_workflows) (effective head), Create invenio_workflows tables.
71634726ec7e (9848d0149abd) -> c25ef2c50ffa (invenio_userprofiles) (head), Create userprofiles tables.
862037093962 -> 07fb52561c5c (invenio_records) (head), Alter column from json to jsonb.
2da9a03b0833 (2e97565eba72, 862037093962) -> 1ba76da94103 (default, invenio_records_files) (head), Create records files tables.
f615cee99600 -> 999c62899c20 (invenio_pidstore) (head), Create pidstore tables.
97bbc733896c -> bff1f190b9bd (default, invenio_oauthclient) (head), Add Timestamp to oauthclient tables.
2e97565eba72 -> f741aa746a7d (invenio_files_rest) (head), Alter files_files.last_check column to nullable.
9848d0149abd -> e12419831262 (invenio_accounts) (head), Add new columns on SessionActivity.
dbe499bfda43 -> 04480be1593e (invenio_access) (head), Add ActionSystemRoles.
eaab22c59b89 -> 2f5368ff6d20 (inspirehep) (head), Add index on legacy_records_mirror
```